pkg/vm/vm_test.go

```diff
@@ -1050,49 +990,35 @@ func getBigInt(a, b int64) *big.Int {
 	return p
 }
 
-func TestAdd(t *testing.T) {
-	prog := makeProgram(opcode.ADD)
-	vm := load(prog)
-	vm.estack.PushVal(4)
-	vm.estack.PushVal(2)
-	runVM(t, vm)
-	assert.Equal(t, int64(6), vm.estack.Pop().BigInt().Int64())
+func TestArith(t *testing.T) {
+	// results for 5 and 3
+	testCases := map[opcode.Opcode]int{
+		opcode.ADD: 8,
+		opcode.MUL: 15,
+		opcode.DIV: 1,
+		opcode.MOD: 2,
+		opcode.SUB: 2,
```

Comment:
Given this test set we can replace MOD with SUB and no one would notice, I'd suggest using some other values.

Reply:
FIxed.